client/src/features/project/projectKg.api.ts

```diff
@@ -26,22 +26,33 @@ import {
   DatasetKg,
   DeleteProjectParams,
   DeleteProjectResponse,
+  EditAvatarProjectParams,
+  EditProjectParams,
   GetDatasetKgParams,
+  KgMetadataResponse,
   ProjectActivateIndexingResponse,
   ProjectIndexingStatusResponse,
+  ProjectKgParams,
+  UpdateProjectResponse,
 } from "./Project";
 import { ProjectIndexingStatuses } from "./projectEnums";
-import { projectsKgApi } from "../projects/projectsKgApi";
+import {
+  ErrorDataMessage,
+  KgJsonLdResponse,
+  ProjectKgContent,
+  ProjectKgWithIdParams,
+} from "./projectKg.types";
 
-interface errorDataMessage {
-  data: {
-    message: string;
+export function kgProjectRequestHeaders(content: ProjectKgContent) {
```

Comment:
It doesn't seem like this needs to be exported.

```suggestion
function kgProjectRequestHeaders(content: ProjectKgContent) {
```